common/app/navigation/NavLinks.scala

```diff
@@ -124,7 +133,7 @@ object NavLinks {
   )
   val soccer = NavLink(
```

Comment:
Yeah we could do this. I left them as they are because soccer is only used in US and wellness front exists only in US edition but I agree this is a bit subtle. I'll update!